data/appRateLimiter_test.go

```diff
@@ -488,7 +488,7 @@ func TestAppRateLimiter_TxgroupToKeys(t *testing.T) {
 }
 
 func BenchmarkAppRateLimiter_TxgroupToKeys(b *testing.B) {
-	rnd := rand.New(rand.NewSource(123))
+	rnd := rand.New(rand.NewPCG(0, 123))
```

Comment:
If you use this: `rnd := rand.NewChaCha8(seed)` you won't have to change the `Read()` calls, I think.